.github/workflows/release.yml

```diff
@@ -0,0 +1,123 @@
+name: Release
+
+env:
+  GO111MODULE: "on"
+  GOPATH: "/home/runner/work/armada/armada/go"
+  GOCACHE: "/home/runner/.cache/go-build"
+
+on: 
+  push:
+    tags: 
+      - 'v*'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-20.04
+    env:
+      GO111MODULE: "on"
+    steps:
+      - name: Checkout code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          path: ${{ env.GOPATH }}/src/github.com/G-Research/armada
+      
+      - name: Build armadactl release artifacts
+        run: |
+          TAG=${GITHUB_REF##*/}
+          make build-armadactl-release RELEASE_VERSION=${TAG}
+        working-directory: ${{ env.GOPATH }}/src/github.com/G-Research/armada
+
+      - name: Archive production artifacts  
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: |
+            ${{ env.GOPATH }}/src/github.com/G-Research/armada/dist/*
+
```

Comment:
I don't see where you publish all the release images, equivalent of:
https://github.com/G-Research/armada/blob/master/.circleci/config.yml#L277

Reply:
my mistake, didn't see diff with images we are using in `build` phase ( `dev` suffix 👨‍🚀 )
added now 🚀 [here ](https://github.com/ljubon/armada/blob/master/.github/workflows/release.yml#L50)and CI output [here ](https://github.com/ljubon/armada/runs/3285852340?check_suite_focus=true#step:6:1)